content/02.main-text.md

```diff
@@ -20,8 +20,23 @@ Although we requested that some authors participate for their specific expertise
 `TODO: confirm "most"`
 To coordinate this effort, we developed a manuscript writing process using the Markdown language, the GitHub software development platform [@url:https://github.com/greenelab/deep-review/], and our new Manubot tool [@url:https://github.com/greenelab/manubot-rootstock; @url:https://github.com/greenelab/manubot] for automating manuscript generation.
 
+### Manubot
 
-### Collaborative writing and reviewing
+We developed Manubot, a system for writing scholarly manuscripts via GitHub.
+With Manubot, manuscripts are written as plain-text markdown files, which is well suited for version control using git.
+The markdown standard itself provides limited yet crucial formatting syntax, including the ability to embed images and format text via bold, italics, hyperlinks, headers, blockquotes, inline code, codeblocks, blockquotes, and numbered or bulleted lists.
+In addition, Manubot relies on Pandoc's extended markdown to enable citations, tables, captions, and equations specified using the popular TeX math syntax.
+
+Manubot includes an additional layer of citation processing, currently unique to the system.
+All citations point to standard identifier, for which Manubot automatically retrieves bibliographic metadata.
+Currently, citations to DOIs (Digital Object Idenfiers), PubMed identifiers, arXiv identifiers, and URLs (web addresses) are supported.
+Metadata is retrieved using DOI [Content Negotiation](https://citation.crosscite.org/docs.html), NCBI's [Citation Exporter](https://www.ncbi.nlm.nih.gov/pmc/tools/ctxp/), the [arXiv API](https://arxiv.org/help/api/index), or [Greycite](http://greycite.knowledgeblog.org/) [@arxiv:1304.7151v1].
```

Comment:
I had been using URL citations instead of links.  We should be consistent.  Links might be best.

Reply:
I think it makes most sense to hyperlink to services or entities. Citations make sense for "citeable work"... things that generally have at least 3 of the following 4 pieces of metadata: authors, title, publisher, or date.

If a service also has a corresponding citable work, such as Greycite, then you can link and cite.

Along these lines, I'm in favor of just hyperlinking to GitHub repos. When referring to Manubot manuscripts (hosted via GH Pages) a citation is appropriate.

@agitter do you agree?

Reply:
@dhimmel Yes, I agree with that.  I created #11 to remind me to convert some of my existing URL citations.